var/spack/repos/builtin/packages/gl2ps/package.py

```diff
@@ -48,6 +58,9 @@ def cmake_args(self):
         if spec.satisfies("platform=darwin"):
             options.append(self.define("CMAKE_MACOSX_RPATH", True))
 
+        if spec.satisfies("generator=ninja"):
```

Comment:
Now that the variant is conditioned on `platform=windows`, this should also be conditioned on `platform=windows`.